framework/decode/vulkan_replay_consumer_base.cpp

```diff
@@ -6244,6 +6249,21 @@ VulkanReplayConsumerBase::OverrideQueuePresentKHR(PFN_vkQueuePresentKHR
                 }
             }
         }
+        const auto* present_fence_info =
```

Comment:
This will only find VkSwapchainPresentFenceInfoEXT if it's the first extension struct in the chain.  Could you descend the chain just to be sure?

Reply:
Not sure if I'm missing something, but `GetPNextMetaStruct` goes through all pNext

Reply:
Nope, you were not missing anything and today I learned something new.